pkg/wireguard/agent/agent.go

```diff
@@ -411,17 +417,14 @@ func (a *Agent) OnIPIdentityCacheChange(modType ipcache.CacheModification, ipnet
 	}
 
 	if updatedPeer != nil {
-		scopedLog := log.WithFields(logrus.Fields{
-			logfields.Modification: modType,
-			logfields.IPAddr:       ipnet.String(),
-			logfields.OldNode:      oldHostIP,
-			logfields.NewNode:      newHostIP,
-			logfields.PubKey:       updatedPeer.pubKey,
-		})
-		scopedLog.Debug("Updating Wireguard peer allowedIPs")
 		if err := a.updatePeerByConfig(updatedPeer); err != nil {
-			scopedLog.
-				WithError(err).
+			log.WithFields(logrus.Fields{
+				logfields.Modification: modType,
+				logfields.IPAddr:       ipnet.String(),
+				logfields.OldNode:      oldHostIP,
+				logfields.NewNode:      newHostIP,
+				logfields.PubKey:       updatedPeer.pubKey,
+			}).WithError(err).
 				Errorf("Failed to update Wireguard peer after ipcache update")
```

Comment:
tiny nit: as there are no printf verbs in the format string, you can use `Error` instead of `Errorf`.

Reply:
Thanks! Fixed.